treeherder/seta/README.md

```diff
@@ -0,0 +1,18 @@
+
```

Comment:
This is really just duplicating the Django (for database migrations) and Treeherder (for running the tests) documentation. It's fine to keep notes for your own records, but I'd rather not put this type of documentation in the tree as it's redundant and tends to go out of date.